nemo/lightning/run/plugins.py

```diff
@@ -215,7 +275,14 @@ def setup(self, task: run.Partial | run.Script, executor: run.Executor):
         logging.info(f"Validating {task.__fn_or_cls__.__qualname__} and {executor.__class__.__qualname__}.")
         if self.validate_preemption:
             logging.info("Validating preemption callback")
-            assert any(map(lambda callback: callback.__fn_or_cls__ == PreemptionCallback, task.trainer.callbacks))
+            assert any(
+                map(
+                    lambda callback: (
+                        callback.__fn_or_cls__ == PreemptionCallback if '__fn_or_cls__' in dir(callback) else False
```

Comment:
Why does Nsys not work with Preemption? is this supposed to be FaultToleranceCallback ?